src/components/common/field/Field.tsx

```diff
@@ -28,9 +35,25 @@ const defaultProps = {
   variant: FieldVariant.DEFAULT,
 };
 
-const Field = ({ field, onChange, value, variant }: Props) => {
+const Field = ({ field, isEditing, onChange, value, variant }: Props) => {
   const classes = useStyles();
   const id = `${field.role} ${field.name}`;
+
+  if (!isEditing) {
+    return (
```

Comment:
Glad you decide to do this instead of a turnery, because it would've been kinda gross to read 😆